internal/adapters/terraform/kubernetes/adapt.go

```diff
@@ -1,10 +1,15 @@
 package kubernetes
 
 import (
+	"regexp"
+	"strings"
+
 	"github.com/aquasecurity/defsec/pkg/providers/kubernetes"
 	"github.com/aquasecurity/defsec/pkg/terraform"
 )
 
+var versionRegex = regexp.MustCompile(`^v\d+(beta\d+)?$`)
```

Comment:
what about `alpha` resources? terraform says this about them https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/guides/versioned-resources#alpha-resources

Reply:
Provider withdraws support for alpha version:
> We will continue our policy of not building support for alpha versioned resources into the provider